src/main/java/com/zufar/onlinestore/payment/exception/UnexpectedPaymentStatusException.java

```diff
@@ -1,15 +1,15 @@
 package com.zufar.onlinestore.payment.exception;
 
-import com.zufar.onlinestore.payment.enums.PaymentStatus;
+import com.zufar.onlinestore.payment.enums.PaymentConstants;
 import lombok.Getter;
 
 @Getter
 public class UnexpectedPaymentStatusException extends RuntimeException {
 
-    private final PaymentStatus paymentStatus;
+    private final PaymentConstants paymentConstants;
 
-    public UnexpectedPaymentStatusException(final PaymentStatus paymentStatus) {
-        super(String.format("Payment status: %s is unexpected.", paymentStatus));
-        this.paymentStatus = paymentStatus;
+    public UnexpectedPaymentStatusException(final PaymentConstants paymentConstants) {
+        super(String.format("Payment status: %s is unexpected.", paymentConstants));
```

Comment:
`"Payment status = %s is unexpected", paymentStatus)`

Reply:
Done